cypress/integration/login.js

```diff
@@ -8,20 +8,28 @@ describe('Sourcebooks login', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    it('Should be able to login with role Team Lead', function () {
 
         cy.visit('/');
         cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
+        cy.get('[aria-label="Valentinas Kasteckis"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="User"]').click();
+        cy.get('[aria-label="Team Lead"]').click();
         cy.get('[type="submit"]').click();
 
         cy.url().should('include', '/time-logging');
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
-        cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.user-info__title').contains('Valentinas Kasteckis');
+        cy.get('.main-nav').find('li').should('have.length', 2);
+    })
+
+    it('Date should match todays date', function() {
+        var d = new Date();
+        cy.get('.calendar--selected').find('span').contains(d.getDate());

```

Comment:
Since there are two "span" tags for a given day, it would be better to get the ".calendar__date" :)

Reply:
@linukkis But calendar__date is always at the top and find() method returns first selected.

Reply:
Yes, but imagine that changed, e.g. a new "span" tag appeared above the date. I understand what you're saying and sometimes we indeed have no other choice other than select the first element (e.g. when all children nodes have no distinct classes or attributes and are very similar), but if we can distinctively retrieve the element we want, it would be better to do it

Reply:
@linukkis I did what you suggested, but @DavidV44 said ".get(".calendar__date") is not needed for this to work correctly, .calendar--selected is enough to find correct element".

Reply:
I'll have a word with him :D It's just that if you write `cy.get('.calendar--selected').contains(d.getDate());` and want to see if the selected day is 1, but the selected day is actually 2 with 1 hour logged, I'm almost sure the test would pass even though the date is incorrect, because it would assert the logged hours as well